applications/asset_tracker/Kconfig

```diff
@@ -18,6 +18,13 @@ config CLOUD_BUTTON_INPUT
 	range 1 1 if BOARD_NRF9160_PCA20035NS
 	default 1
 
+config CLOUD_WAIT_DURATION
+	int "Cloud connection acknowledge wait duration"
+	default 30
+	help
+	    How long the device should wait for a CONNACK
```

Comment:
Other help entries use tabulation for the help description indent. AFAIK Zephyrs standard is two spaces for the indentation of the help message, but at least we should be consistent with other entries in this file.

You could also mention, that the value is in seconds.

Reply:
Fixed.